### PR TITLE
add/tweak some references

### PR DESCRIPTION
**Pull request by davegarrett**
Just noticed that the MTI suites section didn't link to the suites, so here's a minor PR with some reference fiddling for this section:
* Add a link in MTI suites to the suites section, now that they're defined in this document.
* Nitpick ordering of extensions in MTI extensions to have them in same order as sections (so numbers in links are consistient).
* Change plaintext reference to appendices in Security Considerations to actual references.

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -3993,11 +3993,11 @@ In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the following TLS extensions:
 
   * Supported Versions ("supported_versions"; {{supported-versions}})
+  * Cookie ("cookie"; {{cookie}})
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: Why did you move this?

File: draft-ietf-tls-tls13.md [hunk 2]
```diff
@@ -3979,7 +3979,7 @@ all exporter computations.
 In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the TLS_AES_128_GCM_SHA256
 cipher suite and SHOULD implement the TLS_AES_256_GCM_SHA384 and
-TLS_CHACHA20_POLY1305_SHA256 cipher suites.
+TLS_CHACHA20_POLY1305_SHA256 cipher suites. (see {{cipher-suites}})
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: This seems unnecesary.

File: draft-ietf-tls-tls13.md (hunk 1 above)
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: https://tools.ietf.org/html/draft-ietf-tls-tls13-17#section-8.2

When linkified, it currently goes Section 4.2.1, 4.2.3, 4.2.4, 4.2.5, 4.2.6, 4.2.2. Just putting them in order. (as I said, just a nitpick)

File: draft-ietf-tls-tls13.md (hunk 2 above)
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: They're bran new suites, so to someone first reading this document, nothing about that has really been stated as of yet. It occurred to me that could be confusing, hence the PR just to add a ref here. It's a big spec; cross-referencing can help, where applicable.
